my-app/src/App/components/Multidropdown/MultiDropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+import { MultiDropdownProps } from "@config/const";
+
+import styles from "./MultiDropdown.module.scss";
+
+const MultiDropdown: React.FC<MultiDropdownProps> = ({
+  value,
+  onChange,
+  ...props
+}) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  return (
+    <div className={styles.multiDropdown}>
+      <button
+        className={styles.multiDropdown__button}
+        disabled={props.disabled}
+        onClick={() => {
+          setIsOpen(!isOpen);
```

Comment:
когда для сеттинга стейта нужно ориентироваться на его предыдущее значение лучше использовать коллбэк
```suggestion
          setIsOpen(prevValue => !prevValue);
```